api/maintainer/get-pending-payout.js

```diff
@@ -0,0 +1,23 @@
+const { MSGS: { INTERNAL_SERVER_ERROR } } = require('../../helpers/constants')
+
+module.exports = async (req, res, ctx) => {
+  try {
+    ctx.log.info('fetching pending payout for %s', req.session.userId)
+
+    let payout = 0
+    const payoutRes = await ctx.db.maintainer.getPendingPayout({
+      maintainerId: req.session.userId
+    })
+
+    if (payoutRes.length) {
+      // Round payout to the nearest 10 cents to obfuscate ad revenue abuse mechanism
+      payout = (payoutRes.reduce((total, { payout: p }) => total + p, 0) / 100000).toFixed(1)
+    }
+
+    res.send({ success: true, payout })
```

Comment:
It looks like payout is sent back whether or not payoutRes is empty. So can L14 be moved out of the if block, and payout be defined once? reduce works on empty arrays, and you are giving it a starting value of 0 anyway, so it should all work out.